server/app/models/person.rb

```diff
@@ -24,6 +24,7 @@ class Person < ActiveRecord::Base
   has_attached_file :image
 
   has_many :castings, dependent: :destroy
+  belongs_to :listable, polymorphic: true
```

Comment:
Ok! I've always saw people using "something-able" to refer to polymorphic associations, but I'll change it :)